rust/gui-client/src-tauri/src/settings.rs

```diff
@@ -39,6 +53,26 @@ pub(crate) async fn reset_advanced_settings(
     Ok(())
 }
 
+#[derive(Clone, Default, Debug)]
+pub struct MdmSettings {
+    pub auth_url: Option<Url>,
+    pub api_url: Option<Url>,
+    pub log_filter: Option<String>,
+    pub account_slug: Option<String>,
+    pub internet_resource_enabled: Option<bool>,
+    pub hide_admin_portal_menu_item: Option<bool>,
+    pub connect_on_start: Option<bool>,
+    pub start_on_login: Option<bool>,
+    pub disable_update_check: Option<bool>,
+    pub support_url: Option<Url>,
```

Comment:
I tested all of these by manually setting the corresponding registry keys. A few of them are also unit-tested, e.g. the ones affecting the tray menu.